packages/@webex/plugin-meetings/test/unit/spec/roap/index.ts

```diff
@@ -47,8 +47,12 @@ describe('Roap', () => {
       correlationId: 'correlation id',
       selfUrl: 'self url',
       mediaId: 'media id',
-      isAudioMuted: () => true,
-      isVideoMuted: () => false,
+      audio:{
+        isLocallyMuted: () => true,
+      },
+      video:{
+        isLocallyMuted: () => true,
```

Comment:
previously the test was setup for isVideoMuted to return false, probably so that we check that the video and audio are not mixed up, might be worth keeping it that way

Reply:
Sorry I missed it out. Corrected now. Thanks @marcin-bazyl